controllers/genres.js

```diff
@@ -0,0 +1,38 @@
+const models = require('../models')
+
+const getAllGenres = async (request, response) => {
+  try {
+    const allGenres = await models.Genres.findAll()
+
+    return response.send(allGenres)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve genre list, please try again.')
+  }
+}
+
+const getGenreById = async (request, response) => {
```

Comment:
This route throws an error

![image](https://user-images.githubusercontent.com/5775716/82481236-0441f680-9aa3-11ea-9821-557c26f19772.png)